packages/react-broadcast/typings/index.d.ts

```diff
@@ -2,5 +2,10 @@
 // TypeScript Version: 2.x
 
 declare module '@flopflip/react-broadcast' {
+  import Flag from '@flopflip/types';
   export * from '@flopflip/types';
+
+  export interface ToggleComponentProps extends Flag {
+    isFeatureEnabled?: boolean;
```

Comment:
Sorry for the late response. I don't think this is correct. `ToggleComponent` receives a `flag` and a `variation` this `variation prop goes through `isFeatureEnabled` which "translates" it to a `boolean` which then hits `ToggleComponent` in the react package. This way we can share most of the component's logic.